src/appleseed/renderer/kernel/rendering/pixelrendererbase.cpp

```diff
@@ -115,6 +109,7 @@ void PixelRendererBase::on_tile_end(
 
                 switch (sample_state[0])
                 {
+                  default:
```

Comment:
Why this change? What other sample states can there be?

Reply:
Because I forgot to clear the tile before all. It will be removed when the AOV branch will be merged.